pkgs/applications/misc/audio/sox/default.nix

```diff
@@ -32,13 +35,15 @@ stdenv.mkDerivation rec {
     optional enableLibmad libmad ++
     optionals enableLibogg [ libogg libvorbis ] ++
     optional enableFLAC flac ++
-    optional enablePNG libpng;
+    optional enablePNG libpng ++
+    optional enableLibsndfile libsndfile ++
+    optionals enableAMR [ amrnb amrwb ];
 
   meta = {
     description = "Sample Rate Converter for audio";
     homepage = http://sox.sourceforge.net/;
     maintainers = [ lib.maintainers.marcweber ];
-    license = lib.licenses.gpl2Plus;
+    license = if enableAMR then lib.licenses.unfree else lib.licenses.gpl2Plus;
```

Comment:
I'm not sure of this. Dependency on those libs doesn't change the license of sox itself, does it?

Side note: Debian seems to contain free versions of amr* https://packages.debian.org/stretch/libopencore-amrnb0.